integration_tests/e2e/refer.cy.ts

```diff
@@ -275,6 +275,44 @@ context('Refer', () => {
     taskListPage.shouldHaveConfirmedOasys()
   })
 
+  it('Links to the check answers page when the referral is ready for submission', () => {
+    cy.signIn()
+
+    const course = courseFactory.build()
+    const courseOffering = courseOfferingFactory.build()
+
+    const prisoner = prisonerFactory.build({
+      firstName: 'Del',
+      lastName: 'Hatton',
+    })
+    const person = personFactory.build({
+      currentPrison: prisoner.prisonName,
+      name: 'Del Hatton',
+      prisonNumber: prisoner.prisonerNumber,
+    })
+
+    const prison = prisonFactory.build({ prisonId: courseOffering.organisationId })
+    const organisation = OrganisationUtils.organisationFromPrison('an-ID', prison)
+
+    const referral = referralFactory.build({
+      oasysConfirmed: true,
+      offeringId: courseOffering.id,
+      prisonNumber: person.prisonNumber,
+    })
+
+    cy.task('stubCourseByOffering', { course, courseOfferingId: courseOffering.id })
+    cy.task('stubCourseOffering', { courseId: course.id, courseOffering })
+    cy.task('stubPrison', prison)
+    cy.task('stubPrisoner', prisoner)
+    cy.task('stubReferral', referral)
+
+    const path = referPaths.show({ referralId: referral.id })
+    cy.visit(path)
+
+    const taskListPage = Page.verifyOnPage(TaskListPage, { course, courseOffering, organisation, referral })
+    taskListPage.shouldBeReadyForSubmission()
```

Comment:
Could we have a corresponding expectation for the taskListPage not being ready for submission? We could just add an expectation to the first "Shows the in-progress referral task list" page, if you'd rather not add a whole new "it" block.

Reply:
We could - it's kind of tested by `shouldContainTaskList` when the stubbed referral has no completed fields, but I guess it would be good to have a more explicit assertion on this section/task for safety?

Reply:
I think it'd be good to have the explicit assertion, yeah